.github/dependabot.yml

```diff
@@ -185,3 +185,9 @@ updates:
       # TODO: Remove when we drop support for Python 3.10
       - dependency-name: "tomli"
         update-types: [ "version-update:semver-major" ]
+  - package-ecosystem: "docker"
+    directory: "/gradle"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "16:00"
```

Comment:
Sample pull request https://github.com/yeikel/dependabot-core/pull/84/files

We can consider limiting this to only minor or patch upgrades. I am open for feedback